pkgs/top-level/all-packages.nix

```diff
@@ -807,7 +807,7 @@ with pkgs;
 
   borgbackup = callPackage ../tools/backup/borg { };
 
-  boomerang = callPackage ../development/tools/boomerang { };
+  boomerang = qt5.callPackage ../development/tools/boomerang { };
```

Comment:
Please use `libsForQt5.callPackage`.

Reply:
I somehow got it into my head that qt5.callPackage was no preferred over libsForQt5; I'm not sure where this came from, and will of course fix it :).